API_README.md

```diff
@@ -621,7 +621,155 @@ For Gantt chart data specific to a trace ID, modify the request body accordingly
             "timestamp": 1701745060056
         }
 
+## 5. Gantt Chart Data
+    endpoint: api/traces/ganttchart
+    method: POST
 
+    Example:
+    request: http://localhost:80/api/traces/ganttchart
+    body:
+        {
+            "startEpoch": "now-1h",
+            "endEpoch": "now",
+            "searchText": "trace_id=95db2d5796f8986dbeccec3d1582ee85"
+        }
+    response:
+        {   
+            {
+                "span_id": "a98166653c6f7f44",
+                "start_time": 1702310799070961400,
+                "end_time": 1702310799073050400,
+                "duration": 2088875,
+                "service_name": "featureflagservice",
+                "operation_name": "/",
+                "tags": {
+                    "_index": "traces",
+                    "app.ads.ad_request_type": null,
+                    "app.ads.ad_response_type": null,
+                    "app.ads.category": null,
+                    "app.ads.contextKeys": null,
+                    "app.ads.contextKeys.count": null,
+                    "app.ads.count": null,
+                    "app.cart.items.count": null,
```

Comment:
A few things for the `tags`:
1. You can remove the `_index` since that's just from siglens and not part of the original span. I think you can also remove the `timestamp` below (line 760) for the same reason, but double check me on that
2. You should remove all the tags whose value is null.

Also update the code to make sure these changes happen when you actually call the endpoint